packages/commons-test/src/testUtils.ts

```diff
@@ -92,6 +93,23 @@ export function sleep(ms: number): Promise<void> {
   return new Promise((resolve) => setTimeout(resolve, ms));
 }
 
+export const flushPDFMetadata = async (
+  byteArray: Uint8Array,
+  currentExecutionTime: Date
+): Promise<Uint8Array> => {
+  const pdfModified = await PDFDocument.load(byteArray);
+  // Remove metadata properties
+  pdfModified.setTitle("");
+  pdfModified.setAuthor("");
+  pdfModified.setSubject("");
+  pdfModified.setKeywords([]);
+  pdfModified.setProducer("");
+  pdfModified.setCreator("");
+  pdfModified.setCreationDate(currentExecutionTime);
+  pdfModified.setModificationDate(currentExecutionTime);
+  return await pdfModified.save();
+};
+
```

Comment:
Is this still needed?

Reply:
removed in [4ac0d95](https://github.com/pagopa/interop-be-monorepo/pull/1214/commits/4ac0d95eaf780a928374aff4423d9acadc31f2f8)